docs/osquery/osquery.asciidoc

```diff
@@ -82,23 +82,26 @@ You can run packs as live queries or schedule packs to run for one or more agent
 
 . Click the **Packs** tab.
 . Click **Add pack** to create a new pack, or click the name of an existing pack, then **Edit** to add queries to an existing pack.
+. Provide a name for the pack. The short description is optional. 
+. Schedule the pack to be deployed on specified agent policies (*Policy*) or on all agent policies (*Global*).
++
+TIP: Pack deployment details are stored within the <<osquery-custom-config,Osquery configuration>>. The `shard` field value shows the percentage of agents within the policy to which the pack has been deployed.  
```

Comment:
This is better and easier to read. Sometimes I forget that the clearer option is to describe what something is, rather than what something shows or appears to be.